src/OpenGL/Silk.NET.OpenGL.Legacy/Enums/FogParameter.gen.cs

```diff
@@ -12,22 +12,22 @@ namespace Silk.NET.OpenGL.Legacy
     [NativeName("Name", "FogParameter")]
     public enum FogParameter : int
     {
-        [Obsolete("Deprecated in favour of \"Index\"")]
+        [System.Obsolete]
         [NativeName("Name", "GL_FOG_INDEX")]
         FogIndex = 0xB61,
-        [Obsolete("Deprecated in favour of \"Density\"")]
+        [System.Obsolete]
         [NativeName("Name", "GL_FOG_DENSITY")]
         FogDensity = 0xB62,
-        [Obsolete("Deprecated in favour of \"Start\"")]
+        [System.Obsolete]
         [NativeName("Name", "GL_FOG_START")]
         FogStart = 0xB63,
-        [Obsolete("Deprecated in favour of \"End\"")]
+        [System.Obsolete]
         [NativeName("Name", "GL_FOG_END")]
         FogEnd = 0xB64,
-        [Obsolete("Deprecated in favour of \"Mode\"")]
+        [System.Obsolete]
         [NativeName("Name", "GL_FOG_MODE")]
         FogMode = 0xB65,
-        [Obsolete("Deprecated in favour of \"Color\"")]
+        [System.Obsolete]
```

Comment:
It seems your code doesnt put a helpful obsolete message here, returning the obsolescence to what it was before, i feel like my patch's behaviour of always overwriting obsolete messages is something we should try to carry over

Reply:
This was intentional. If a token is obsoleted at the convert/construct level, all variants of that token should be obsoleted with the same message - you should not use that value!

This particular obsolete attribute is coming from OpenGLReader

Reply:
> This was intentional. If a token is obsoleted at the convert/construct level, all variants of that token should be obsoleted with the same message - you should not use that value!
> 
> This particular obsolete attribute is coming from OpenGLReader

I mean i do understand its coming from OpenGLReader and intentional, but if we are able to detect in the bakery the new name and create a more helpful message to the user, imo we should update the message, unless theres a case where the bakery produces a wrong message and doesnt have the proper information on the enum here?

Reply:
I wouldn't object to changing OpenGLReader to add a message.

Reply:
I attempted to do this, but i dont understand ReadEnums in OpenGLReader enough and could not figure out how to get the new name into there

Reply:
Shall we leave this comment then

Reply:
> Shall we leave this comment then

If you dont feel like working on it (i tried again last night and couldnt figure it out again), then its not a blocker, and this discussion can be closed, with maybe a `TODO` or a note in the code marking something to eventually be done, if someone cares

Reply:
Done. Donezo. Donion ring.